### DISCUSSION TITLE
Can we hook into inline blame annotations or the on:hover blame dialog?

### DISCUSSION THREAD

**Discussion by juzerzarif** [gitkraken/vscode-gitlens#2152] [Q&A] [answered]
So for work we follow a convention where all our commit messages start with a JIRA ticket number (like ABC-1234: my commit message). Very often I have to copy the JIRA key from the inline blame info dialog and then go look at that JIRA ticket.

I'm wondering if there's a way I can hook into gitlens's blame annotations (or even better, the dialog) to turn those JIRA keys in the commit messages to hyperlinks.

Screenshot to explain what I'm talking about:
![Screen Shot 2022-08-18 at 10 57 38 AM](https://user-images.githubusercontent.com/22772637/185428202-62061c8a-22cb-49aa-ba93-c30917918a2c.jpg)

**Comment by d13** [comment, marked as answer]
@juzerzarif we have an autolinks feature that can linkify your JIRA tickets.

From the command palette, select `GitLens: Configure Autolinks` to open the autolinks editor in our settings screen:

![Screen Shot 2022-08-18 at 3 09 07 PM](https://user-images.githubusercontent.com/293282/185474998-5786af3e-cdf0-4ef8-90b2-2e6734a9d0f9.png)
:

**Reply by juzerzarif** [reply]
Ah seems I completely missed this feature being a thing. Thanks a lot!
